app/gui2/src/components/GraphEditor/widgets/WidgetCheckbox.vue

```diff
@@ -1,20 +1,20 @@
 <script setup lang="ts">
 import CheckboxWidget from '@/components/widgets/CheckboxWidget.vue'
 import { Tree } from '@/generated/ast'
-import { Score, defineWidget, widgetAst, type WidgetProps } from '@/providers/widgetRegistry'
+import { Score, defineWidget, widgetProps } from '@/providers/widgetRegistry'
 import { useGraphStore } from '@/stores/graph'
-import { Ast, type AstExtended } from '@/util/ast'
+import { AstExtended } from '@/util/ast'
 import { computed } from 'vue'
 
-const props = defineProps<WidgetProps>()
+const props = defineProps(widgetProps(widgetDefinition))
+
 const graph = useGraphStore()
 const value = computed({
   get() {
-    return widgetAst(props.input)?.repr().endsWith('True') ?? false
+    return props.input.repr().endsWith('True') ?? false
```

Comment:
nit: i don't think the `?? false` is necessary here - there is no `?.` anywhere that may produce a nullish value.